src/libicalvcal/vcc.y

```diff
@@ -117,9 +117,7 @@ DFARS 252.227-7013 or 48 CFR 52.227-19, as applicable.
 #include <config.h>
 #endif
 
-#include <string.h>
 #include <stdio.h>
-#include <stdlib.h>
```

Comment:
I wonder if you need stdio.h either.  if you include vcc.h first.  in fact, I would like the include order to be (if possible):
config.h
vcc.h
<standard C headers>

Reply:
Without `#include <stdio.h>` *vcc.c* does compile. I have included also that change.

Reply:
great. thanks